src/main/java/edu/greenblitz/tobyDetermined/commands/NodeToNeighbourCommand.java

```diff
@@ -0,0 +1,102 @@
+package edu.greenblitz.tobyDetermined.commands;
+
+import edu.greenblitz.tobyDetermined.Nodesssss.NodeArm;
+import edu.greenblitz.tobyDetermined.Nodesssss.NodeBase;
+import edu.greenblitz.tobyDetermined.commands.telescopicArm.extender.ExtendToLength;
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Elbow;
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Extender;
+import edu.greenblitz.utils.GBCommand;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class NodeToNeighbourCommand extends GBCommand {
+    private NodeBase nodeBase;
+    private final Extender extender;
+    private final Elbow elbowSub;
+    private NodeArm start;
+    private NodeArm end;
+    private static final double COMBINED_VELOCITY = 2.3;
+    private static final double MAX_EXTENDER_VELOCITY = 1; //In Meters Per Second
```

Comment:
the max speed was intended to make it go slow for safety purposes you should look into slowly increasing them and then deleting them